runtime/wasm/src/asc_abi/mod.rs

```diff
@@ -111,20 +97,131 @@ pub trait AscType: Sized {
 /// be equal to alignment.
```

Comment:
Does this comment need any updates?

Reply:
Good catch, it should not be mentioning a default implementation of `AscType`.